agent/xds/delta.go

```diff
@@ -26,6 +26,15 @@ import (
 	"github.com/hashicorp/consul/logging"
 )
 
+type DeltaRecvResponse int
+
+const (
+	DeltaRecvResponseNack DeltaRecvResponse = iota
+	DeltaRecvResponseAck
+	DeltaRecvNewSubscription
+	DeltaRecvUnknownType
+)
```

Comment:
Are these meant to be exported? Would prefer to keep them imported until necessary outside `xds`